project2/get_ingredients.py

```diff
@@ -0,0 +1,176 @@
+# input "1 cup white sugar"
+# output
+    # ingredient: sugar
+    # quantity: 1
+    # measurement: cup
+    # descriptior: white
+    # preparation: ''
+
+# input "1 cup butter, softened"
+# output
+    # ingredient: butter
+    # quantity: 1
+    # measurement: cup
+    # descriptior: ''
+    # preparation: 'soften'
+
+import re
+import nltk
+from nltk.corpus import stopwords
+from collections import Counter
+from fractions import Fraction
+import numpy as np
```

Comment:
this also needs to be added to the Pipfile. please check out the README if you are unsure how to do this